internal/core/command/controller/messaging/external/external.go

```diff
@@ -124,6 +134,78 @@ func commandQueryHandler(responseTopic string, qos byte, retain bool, dic *di.Co
 	}
 }
 
+func commandRequestHandler(dic *di.Container) mqtt.MessageHandler {
+	return func(client mqtt.Client, message mqtt.Message) {
+		lc := bootstrapContainer.LoggingClientFrom(dic.Get)
+		messageBusInfo := container.ConfigurationFrom(dic.Get).MessageQueue
+		qos := messageBusInfo.External.QoS
+		retain := messageBusInfo.External.Retain
+
+		// expected command request topic scheme: #/<device>/<command-name>/<method>
+		topicLevels := strings.Split(message.Topic(), "/")
+		length := len(topicLevels)
+		deviceName := topicLevels[length-3]
+		commandName := topicLevels[length-2]
+		method := topicLevels[length-1]
+		// expected command response topic scheme: #/<device>/<command-name>/<method>
+		externalResponseTopic := messageBusInfo.External.Topics[ResponseCommandTopicPrefix] + strings.Join([]string{deviceName, commandName, method}, "/")
```

Comment:
does this put a `/` between ResponseCommandTopicPrefix an the rest?
Might be better as:
```suggestion
		externalResponseTopic :=strings.Join([]string{ messageBusInfo.External.Topics[ResponseCommandTopicPrefix] , deviceName, commandName, method}, "/")
```